### DISCUSSION TITLE
播放大文件视频时显示连接失败

### DISCUSSION THREAD

**Discussion by b1tzer** [AlistGo/alist#635] [Q&A] [answered]
播放123Pan里的视频(大小：7.5G)失败，请问这个该怎么解决

Error Log
Feb 28 17:14:24 VM-20-4-ubuntu alist[888]: [GIN] 2022/02/28 - 17:14:24 | 200 |  338.430919ms |  113.87.129.100 | GET      "/p/123Pan/Movie/The.Matrix.Resurrections.2021.1080p.HMAX.WEB-DL.DDP5.1.Atmos.HDR.x265-MZABI.mkv?sign=b8c11430ab127597"
Feb 28 17:14:24 VM-20-4-ubuntu alist[888]: Error #01: write tcp 127.0.0.1:5244->127.0.0.1:54662: write: broken pipe

公网链接：https://0ad.xyz/123Pan/Movie/The.Matrix.Resurrections.2021.1080p.HMAX.WEB-DL.DDP5.1.Atmos.HDR.x265-MZABI.mkv

**Comment by xhofe** [comment, marked as answer]
https://alist-doc.nn.ci/docs/common-question#%E8%A7%86%E9%A2%91%E6%92%AD%E6%94%BE%E4%B8%8D%E4%BA%86

另外：你开启了代理，是使用服务器进行中转的，速度会取决于服务器的带宽。

**Reply by b1tzer** [reply]
尝试关闭代理后，网页端点击播放显示连接失败，IINA显示无法打开文件或流。

**Reply by xhofe** [reply]
我下载了你的原文件，它本身就是损坏的，无法播放。

**Reply by cxw620** [reply]
> 

服务器中转会走服务器上面的代理吗

**Comment by xhofe** [comment]
![image](https://user-images.githubusercontent.com/36558727/156017363-0a865a68-6163-4948-b5ed-f9b21dfd2e18.png)

**Reply by b1tzer** [reply]
确实，我试了一下文件真的损坏了，尴尬-_-!!
